ui/next/app/app.tsx

```diff
@@ -59,10 +58,15 @@ import uiReducer from "./redux/ui";
 import metricsReducer from "./redux/metrics";
 
 import Layout from "./containers/layout";
-import { ClusterMain, ClusterTitle } from "./containers/cluster";
-import { DatabasesMain, DatabasesTitle } from "./containers/databases";
-import { HelpUsMain, HelpUsTitle } from "./containers/helpus";
-import { NodesMain, NodesTitle } from "./containers/nodes";
+import Cluster from "./containers/cluster";
+import ClusterOverview from "./containers/clusterOverview";
+import ClusterEvents from "./containers/clusterEvents";
+import Databases from "./containers/databases";
```

Comment:
OK. In your next PR please add `make -C ui/next test` to circle-test.sh (shard 2) to prevent this from recurring